cmd/deployctl/cmd/build.go

```diff
@@ -369,4 +399,7 @@ func init() {
 	collectCmd.Flags().Bool(NormalizeInterfaceMTUFilterArg, false, "Normalize interface MTU values")
 	collectCmd.Flags().Bool(NormalizeConsoleFilterArg, false, "Normalize serial console attributes")
 	collectCmd.Flags().Bool(MinimalConfigFilterArg, false, "Shorthand notation for adding all available filters")
+	collectCmd.Flags().Bool(AllowDRBDConfigArg, false, "To allow DRBD field from system instances")
+	collectCmd.Flags().Bool(AllowAllFileSystemsArg, false, "false:includes only backup,database,instance and image-conversion fs, True:To allow all storage filesystems from system instances")
```

Comment:
This message is formatting ugly, suggest "Allow all types of filesystems, only back, database, instance and image-conversion are allowed by default"

Reply:
updated as above